About.html creation and edited

nero228 opened the pull request:
I put stuff into about.html and edited the other files needed in the project with my input.

nero228 committed:
Created the project1 folder and the .md files needed

steelbeast committed:
added username

steelbeast committed:
Create steelbeast.md

steelbeast committed:
Added issue and research

steelbeast committed:
Added branch

steelbeast committed:
Added line to use normalize.css as final fail safe

steelbeast committed:
Adding normalize.css file into css folder

Allows any page inside the doc folder to access normalize.css if they reference it in their html document.

steelbeast committed:
Added picture alts

steelbeast committed:
added pull link

kragan13 committed:
Added username

kragan13 committed:
Create kragan13.md

kragan13 committed:
Merge pull request #1 from kragan13/Index.html-updating

All three commits in this pull request have no issues and can be safely merged. Index.html is now slightly improved

kragan13 committed:
Add summary of first merge

kragan13 committed:
Fixed issue with line spacing

kragan13 committed:
Added link to the merge

kragan13 committed:
Update kragan13.md

kragan13 committed:
Added one branching strategy

steelbeast committed:
Finishing write up

steelbeast committed:
Added #2 and info on our strategy

steelbeast committed:
Added link in #2

steelbeast committed:
Added a small issue I had while working

nero228 committed:
Added my requirement to conclusion.md

nero228 committed:
updated the users.md and branching.md

nero228 committed:
Added a basic structure to about.html and added to nero228.md

kragan13 committed:
Update kragan13.md

kragan13 committed:
Update Conclusion.md

kragan13 commented:
Contributors - Kyle Ragan (kragan13), Matthew Coleman (steelbeast), Daniel Bleigh (nero228)

mikeknows commented:
Referencing an issue in your commits on accident.

kragan13 committed:
Update about.html

Added information to the <head> section (mostly stylesheets) by copying over the information from index.html, this at least begins to make it possible to style the page

kragan13 committed:
Update about.html

Added longer description of project, and corrected the header tag that was round the "About" header at the top.

kragan13 committed:
Update index.html

Corrected some typos in the text

kragan13 committed:
Update index.html

Made small correction to text

kragan13 committed:
Update index.html

More typo fixing

kragan13 committed:
Update index.html

Added link to about.html in index.html

kragan13 committed:
Update index.html

Added links to GitHub repos for PearPlanner, R. Sanchez's RaiderPlanner, and G. Dong's RaiderPlanner fork.

kragan13 committed:
Updated about and index html files

Corrected course name in about.html. Added contact section, which was previously "under construction", and provided some relevant links including a link to the list of authors, so credit is easily given now.

kragan13 committed:
Merge pull request #2 from kragan13/html-updates

Html files have been updated and improved in some basic ways. They are ready to be merged with the master branch.

kragan13 committed:
Updated about.html and untitled.css

Moved 'About' header into an actual header tag. Added basic styling to the page so it is more presentable (tried to copy color scheme of index.html, could not find exact colors so I guessed)

kragan13 committed:
Merge pull request #3 from kragan13/More-html-updates

Updated about.html and untitled.css, merging with my master branch and will soon submit request to merge with gzdwsu's master branch
